src/coreComponents/unitTests/linearAlgebraTests/testDofManager.cpp

```diff
@@ -644,7 +642,8 @@ TYPED_TEST_P( DofManagerSparsityTest, TPFA_Full )
     { "pressure",
       FieldLocation::Elem,
       DofManager::Connector::Face,
-      2, makeSparsityTPFA }
+      2, makeSparsityTPFA,
+      { {"mesh", "Level0", {} } } }
```

Comment:
Why wasn't the region included until now?

Reply:
I think the intent was that empty (default-initialized) list of mesh-level-region tuples would mean auto-fill, i.e. include the entire mesh. But looking at the implementation I noticed that this only works correctly for an empty region list (the innermost empty braces), but omitting mesh and level resulted in an empty sparsity pattern, so these tests were silently broken for a while - hard to tell when or how this happened, these tests are pretty old, written in the days when we used to assume a default single mesh and level.